jujugui/static/gui/src/app/assets/css/base.scss

```diff
@@ -181,6 +181,38 @@ input[type="radio"]:not(old):checked + label > .outer > .inner {
   border-radius: 3px;
   background: white;
 }
+
+input[type="checkbox"] {
```

Comment:
I'm not sure if we would have a case where we don't want the checkboxes to appear the same...